docs/modules/tile-converter/cli-reference/tile-converter.md

```diff
@@ -10,125 +10,143 @@
-| help                      | \*                        | \*                        | Show the converter tool options list                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-| validate                  | \*                        |                           | Perform counting of all tiles and tiles with "ADD" type of refinement. Check whether a particular child node fits into the parent one or not. If not, warn about it.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| split-nodes               | \*                        |                           | Prevent to merge similar materials that could lead to incorrect visualization. By default, the converter tries to merge PBR materials to create one node for all primitives in a glTF file. Using this option the converter will generate new node for every PBR material in the glTF file. With this option the resulting layer usually has refinement issues because "leaf" nodes are generated in the middle of the nodes tree                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Nit: try to keep descriptions short by cutting unnecessary words
```suggestion
| split-nodes               | \*                        |                           | The converter will generate new node for every PBR material in the glTF file. Prevent merge of similar materials that could lead to incorrect visualization. By default, the converter tries to merge PBR materials to create one node for all primitives in a glTF file. Note can cause refinement issues because "leaf" nodes are generated in the middle of the nodes tree                                                                                                                                                                                                                                                                                                                                 |
```